data/coalition/coalition missions.txt

```diff
@@ -1392,3 +1392,271 @@ mission "Arachi Cartoonist 2"
+			`	You follow the Saryd back to the gravelly path, and then to the spaceport grounds, where Vellorae thanks him for guiding the two of you to the shrine. He offers his condolences and wishes her a safe journey to the next shrine. He soon disappears into the fog again, and you and Vellorae head back to your ship.`
+				goto end
+			label notgo
+			`	She sighs briefly, and agrees to meet with you later, following the Saryd local deep into the fog stretching out of the city. You head inside your ship, waiting for a few hours until you hear her voice just outside your hatch. You open the hatch, and let her in.`
+			label end
+			`	"To <destination>, we must go now. The Path of Lor Ie Vees, I must take," she says, heading to her bunk to rest.`
+				accept
+	on visit
+		dialog `You land on <planet>, but realize that your escort carrying Vellorae hasn't entered the system yet. Better depart and wait for it.`
+	on complete
+		"coalition jobs" ++
+		payment 69700
+
+mission "Coalition: Pilgrimage 5"
+	landing
+	name "Woodland Thing"
+	description "Bring Vellorae to <destination>, where she will finish her mourning pilgrimage."
+	passengers 1
+	to offer
+		has "Coalition: Pilgrimage 4: done"
+	source "Shadow of Leaves"
+	destination "Cold Horizon"
+	on offer
+		conversation
+			`Vellorae hands you 69,700 credits as you land, and says she'll head to the "path" once the sun sets. "Come to this world for our honeymoon, Adfices and I did, and visited the path, we had. Know it's starting point, I already do."`
+			`	She tells you she'll pass time until then by visiting local shops, asking if you'd like to come with her.`
+			choice
+				`	(Go along with her.)`
+				`	(Stay in your ship.)`
+					goto notgo
+			`	She takes you around the spaceport city, touring some clothing shops and bringing you to some parks she tells you she had visited with her husband. Shortly after noon you two enter a restaurant, where she insists on paying for your meal. You tour the city with her for a while longer and, just before the sun sets, reach the forest at the base of the mountain on top of which Starlit University is located.`
+			`	Once night falls, many lights start to appear inside the forest, and Vellorae beckons you to follow her into the forest. "Stick close, you should. Difficult to navigate the path, for other species it often is." A you walk down a trail with her, you approach one of the lights to see that it's a bonfire on a platform in a miniature clearing, at what you hope is a safe distance from the trees around it. Continuing to the next light, Vellorae clings the bangle in her hands the entire time, humming some guttural, almost tribal chant. As you pass by another fire, a short burst of flames gets closer to some nearby shrubs, but thankfully is put out before it can cause any harm, extinguished with an odd, large fabric net on a pole by a cloaked Saryd who seemed to materialize from the trees near the fire.`
```

Comment:
```suggestion
			`	Once night falls, many lights start to appear inside the forest, and Vellorae beckons you to follow her into the forest. "Stick close, you should. Difficult to navigate the path, for other species it often is." As you walk down a trail with her, you approach one of the lights to see that it's a bonfire on a platform in a miniature clearing, at what you hope is a safe distance from the trees around it. Continuing to the next light, Vellorae clutches the bangle in her hands the entire time, humming some guttural, almost tribal chant. As you pass by another fire, a short burst of flames gets closer to some nearby shrubs, but thankfully is put out before it can cause any harm, extinguished with an odd, large fabric net on a pole by a cloaked Saryd who seemed to materialize from the trees near the fire.`
```